index.js

```diff
@@ -209,7 +206,8 @@ export default class Carousel extends Component {
     this.animateToPage(this._normalizePageNumber(currentPage + 1));
   }
 
-  animateToPage = (page) => {
+  animateToPage = (page, options) => {
```

Comment:
I just realized that in the final solution, `options` isn't being used anymore so I'll remove it.